platform/nano/api.go

```diff
@@ -0,0 +1,79 @@
+package nano
+
+import (
+	"strconv"
+
+	"github.com/trustwallet/blockatlas/coin"
+	"github.com/trustwallet/blockatlas/pkg/blockatlas"
+
+	"github.com/spf13/viper"
+)
+
+type Platform struct {
+	client Client
+}
+
+func (p *Platform) Init() error {
+	p.client = Client{blockatlas.InitClient(viper.GetString("nano.api"))}
+	return nil
+}
+
+func (p *Platform) Coin() coin.Coin {
+	return coin.Coins[coin.NANO]
+}
+
+func (p *Platform) GetTxsByAddress(address string) (blockatlas.TxPage, error) {
+	var normalized []blockatlas.Tx
+	history, err := p.client.GetAccountHistory(address)
+
+	if err != nil {
+		return nil, err
+	}
+
+	for _, srcTx := range history.History {
+		tx := p.Normalize(&srcTx, history.Account)
+		normalized = append(normalized, tx)
+	}
+
+	return normalized, nil
+}
+
+func (p *Platform) Normalize(srcTx *Transaction, account string) (tx blockatlas.Tx) {
+	var from string
+	var to string
+	var direction blockatlas.Direction
+
+	if srcTx.Type == "send" {
+		direction = blockatlas.DirectionOutgoing
+		from = account
+		to = srcTx.Account
+	} else if srcTx.Type == "receive" {
+		direction = blockatlas.DirectionIncoming
```

Comment:
i don't think we need to define direction here, as it will be defined somewhere else i believe